core/templates/loginflow/authpicker.php

```diff
@@ -36,8 +36,8 @@
 		])) ?>
 	</p>
 
-	<p class="info">
-		<?php print_unescaped($l->t('If you are not trying to set up a new device or app, someone is trying to trick you into granting them access to your data. In this case do not proceed and instead contact your system administrator.')) ?>
+	<p class="warning">
+		<?php print_unescaped($l->t('<strong>Security warning:</strong> If you are not trying to set up a new device or app, someone is trying to trick you into granting them access to your data. In this case do not proceed and instead contact your system administrator.')) ?>
```

Comment:
```suggestion
		<strong><?php p($l->t('Security warning:')); ?></strong><?php p($l->t('If you are not trying to set up a new device or app, someone is trying to trick you into granting them access to your data. In this case do not proceed and instead contact your system administrator.')) ?>
```
Would keep the old translations instead of duplicating them, same in the other file

Reply:
I was considering the same first but wondered if this may cause issues with RTL languages?  (e.g. arabic)

Reply:
then make it a h3?

Reply:
Done - fair point :)